### ISSUE TITLE
gentrl 0.1 require torch==1.0.1, but molsets require torch==1.1.0

### ISSUE THREAD

**Issue by techstone** [insilicomedicine/GENTRL#1]
gentrl 0.1 has requirement torch==1.0.1, but you'll have torch 1.1.0 which is incompatible

**Comment by danpol** [comment]
Thank you for reporting it! We recently updated torch in MOSES 0.2 to 1.1.0.
For compatibility in GENTRL, we now [specify](https://github.com/insilicomedicine/GENTRL/commit/f01c55bc012e49929684cd976ca400b99fdc6c4b) version MOSES 0.1.3 in GENTRL which uses torch 1.0.1.

**Comment by ghost** [comment]
Hi all,
Here's an error while installing MOSES

**ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

gentrl 0.1 requires molsets==0.1.3, but you'll have molsets 0.3.1 which is incompatible.
gentrl 0.1 requires torch==1.0.1, but you'll have torch 1.1.0 which is incompatible.**

**Comment by ghost** [comment]
Hi there, 

I've overcome the error with new file you have created.
Now how to start with the following steps.

4. Run the pretrain.ipynb to train an autoencoder
5. Run the train_rl.ipynb to optimize a reward function
